Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/PrimerHeadlessUniversalCheckout.swift

```diff
@@ -123,6 +123,10 @@ public class PrimerHeadlessUniversalCheckout: LogReporter {
             }
         }
     }
+    
+    public func setupPaymentIntent(_ intent: PrimerSessionIntent) {
```

Comment:
[question]: Do we want to have this as a public method? cc @NQuinn27

Reply:
Alright, i will remove the public exposure of shared intent and implement the check in KlarnaHeadlessManager as per `private func validatePaymentMethod(withType paymentMethodType: String, andIntent intent: PrimerSessionIntent? = nil) throws -> PrimerPaymentMethod` from `NativeUIManager`